scripts/cmd/dependencies.go

```diff
@@ -8,16 +8,16 @@ import (
 func updateDeps(ctx *cli.Context) error {
 	cfg := &dependencies.Config{
 		Images: &dependencies.ImagesConfig{
-			Names:          []string{"hermeznetwork/geth-zkevm-contracts", "hermeznetwork/zkevm-mock-prover"},
+			Names:          []string{"hermeznetwork/geth-zkevm-contracts", "hermeznetwork/zkprover-local"},
```

Comment:
Maybe we need to use the new 0xPolygonHermez instead fo hermeznetwork

Reply:
Well, these are docker image names as found in docker-compose.yml. Not sure if there's a new `0xPolygonHermez` in Docker Hub, in any case, the image names here should match what we have in docker-compose.yml (as they do now), and both should be updated at the same time.